bindata/network/openshift-sdn/001-crd.yaml

```diff
@@ -64,46 +122,91 @@ spec:
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        egressCIDRs:
+          description: EgressCIDRs is the list of CIDR ranges available for automatically
+            assigning egress IPs to this node from. If this field is set then EgressIPs
+            should be treated as read-only.
+          items:
+            type: string
+            pattern: '^(([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])/([0-9]|[12][0-9]|3[0-2])$'
+          type: array
+        egressIPs:
+          description: EgressIPs is the list of automatic egress IP addresses currently
+            hosted by this node. If EgressCIDRs is empty, this can be set by hand;
+            if EgressCIDRs is set then the master will overwrite the value here with
+            its own allocation of egress IPs.
+          items:
+            type: string
+            pattern: '^(([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])$'
+          type: array
         host:
+          description: Host is the name of the node. (This is the same as the object's
+            name, but both fields must be set.)
           type: string
           pattern: '^[a-z0-9.-]+$'
         hostIP:
+          description: HostIP is the IP address to be used as a VTEP by other nodes
+            in the overlay network
           type: string
           pattern: '^(([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])$'
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
         subnet:
+          description: Subnet is the CIDR range of the overlay network assigned to
+            the node for its pods
           type: string
           pattern: '^(([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])/([0-9]|[12][0-9]|3[0-2])$'
-        egressIPs:
-          type: array
-          items:
-            type: string
-            pattern: '^(([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])$'
-        egressCIDRs:
-          type: array
-          items:
-            type: string
-            pattern: '^(([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])/([0-9]|[12][0-9]|3[0-2])$'
+      required:
+      - host
+      - hostIP
+      - subnet
```

Comment:
And this has the opposite problem; `HostSubnet`'s `subnet` field should not be required